src/tk/shanebee/lesspillagers/LessPillagers.java

```diff
@@ -4,66 +4,100 @@
 import org.bukkit.ChatColor;
 import org.bukkit.command.Command;
 import org.bukkit.command.CommandSender;
+import org.bukkit.configuration.ConfigurationSection;
 import org.bukkit.entity.Entity;
 import org.bukkit.entity.EntityType;
 import org.bukkit.entity.Player;
 import org.bukkit.event.EventHandler;
+import org.bukkit.event.HandlerList;
 import org.bukkit.event.Listener;
 import org.bukkit.event.entity.CreatureSpawnEvent;
 import org.bukkit.plugin.java.JavaPlugin;
 
-import java.util.List;
-
 public class LessPillagers extends JavaPlugin implements Listener {
+  private int MAX_PILLAGERS, RADIUS_X, RADIUS_Y, RADIUS_Z;
+
+  // -- JavaPlugin methods --------------------------------------------------------------------------------------------
+
+  @Override
+  public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
```

Comment:
Oh one more thing, for consistency sake (many devs do this) the command should be after the onEnable.
I prefer this order:
fields
onEnable (should be at the top)
onDisable
commands
everything else